gamecenter/templates/user/footer.html

```diff
@@ -0,0 +1,12 @@
+<footer class="mdl-mini-footer">
+  <div class="mdl-mini-footer__left-section">
+    <div class="mdl-logo">Rank</div>
+    <ul class="mdl-mini-footer__link-list">
+      <li><a href="{{ url_for('public.about') }}">About</a></li>
+      <li><a href="mailto:rdbaker@wpi.edu">Contact</a></li>
```

Comment:
yeah I'll switch this to Heineman's email, because he'll own this after we graduate